README.md

```diff
@@ -104,6 +104,7 @@ Make sure that the secrets we set earlier are in your environment (replace .bash
 
 Prepare the database
 
```

Comment:
Note that this must be run in the root directory. Following along with the readme still has the user in `/docker`.